serviceUsage.html

```diff
@@ -345,8 +362,79 @@ <h2>
                 <td>When specified message protocol is not loaded</td>
             </tr>
         </table>
-
-    </section>
+		<h2>
+            <a id="examples" class="anchor" href="#examples" aria-hidden="true">
+                <span aria-hidden="true" class="octicon octicon-link"></span>
+            </a>
+        </h2>
+		<h2> Lookups </h2>
+		<p> Presently, to generate Eiffel messages, Eiffel REMREM-Generate is being used. This process requires input mapping exactly one-to-one to Eiffel message Schema.</p>
+		<p> The "Link" field of Eiffel Message requires UUIDs of earlier sent events to link to. But these UUIDs are not always available .</p>
+		<p> In order to resolve this , we need to provide LOOKUPS in Links field ,which will have eventType and Properties. eventType will contain type of event.</p>
+		<p> Properties will contain any field of event mentioned in eventType or RepoUri or artifact identity (Purl identity including version) to Eiffel ,</p>
```

Comment:
properties will help to prepare the Query to ER and Lookup the ER data.
Example:
As a User, I want to place the LINK type ARTIFACT to Event ID of EiffelArtifactCreatedEvent from Event Repository.
The user passes the below information to REMReM Generate, REMReM Generate will Query to configured Event Repository and place the event ID in LINK's and generate the Eiffel message.
"links":[
					{
			"type": "ARTIFACT",
			"%lookup%": {
			"eventType": "EiffelArtifactCreatedEvent",
			"properties": [
				{ “data.identifier": "some purl identifier defining the artifact" }]
					}
				}]
ER Query look like: https://<host:port>/eventrepository/events/?eventType=EiffelArtifactCreatedEvent&data.identifier=some purl identifier defining the artifact